### PR TITLE
Feat/add payload to non message top content type #BOT-573

### PR DESCRIPTION
**Pull request by AinaVendrell**
## Description
- Add payload content

## Context
- The flow builder needs to read all the available payloads and its ids

## Testing

The pull request...

- has unit tests

### CODE REVIEW

File: packages/botonic-plugin-contentful/src/cms/cms-dummy.ts [hunk 1]
```diff
@@ -116,6 +117,12 @@ export class DummyCMS implements CMS {
     )
   }
 
+  payload(id: string, {} = DEFAULT_CONTEXT): Promise<Payload> {
+    return Promise.resolve(
+      new Payload(new CommonFields(id, id), 'Dummy payload for ' + id)
```
Comment (vanbasten17) [inline packages/botonic-plugin-contentful/src/cms/cms-dummy.ts:122]: encapsulate the logged id like: `Dummy payload for '${id}'`

**Review by vanbasten17 — APPROVED** [review]
LGTM!!

File: packages/botonic-plugin-contentful/src/contentful/contents/payload.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      id,
+      context
+    )
+    return this.fromEntry(entry, context)
+  }
+
+  async fromEntry(entry: contentful.Entry<PayloadFields>, context: Context) {
+    return addCustomFields(
+      new cms.Payload(
+        await this.getFollowUp().commonFields(entry, context),
```
Comment (asastre) [inline packages/botonic-plugin-contentful/src/contentful/contents/payload.ts:25]: Why obtaining the common fields of the followUp? Shouldn't this be `ContentfulEntryUtils.commonFieldsFromEntry(entry)`?

**Review by asastre — COMMENTED** [review]
Just one comment before approving

**Review by asastre — APPROVED** [review]

**Pull request merged**